### PR TITLE
Add first v2.3 RC

### PR DESCRIPTION
**Pull request by caseydavenport**

### CODE REVIEW

File: _data/versions.yml [hunk 1]
```diff
@@ -289,6 +289,46 @@ v2.2:
     networking-calico:
       version: 1.4.2
       url: http://git.openstack.org/cgit/openstack/networking-calico/commit/?h=1.4.2
+v2.3:
```
Comment (robbrockbank) [inline _data/versions.yml]: Any reason this isn't at the top?

**Comment by robbrockbank** [comment]
LGTM - one minor comment about ordering of the versions file (I think 2.3 should be at the top)

File: _data/versions.yml (hunk 1 above)
Comment (tmjd) [inline _data/versions.yml]: To keep it from being the 'current release'.

Comment (robbrockbank) [inline _data/versions.yml]: Ah gotcha - I assume we also disable it being in the versions drop down no?

Comment (tmjd) [inline _data/versions.yml]: I'm not sure what has to be done to keep it from being there.  Check the RELEASING.md, it goes through that.

**Pull request merged**